.gitignore

```diff
@@ -9,7 +9,7 @@ Thumbs.db
 # ==========================
 .nodemonignore
 .sass-cache/
-npm-debug.log
+npm-debug.log*
```

Comment:
I am not 100% sure now npm handles debug logs but when testing I had 2 different logs and their names where (for example) npm-debug.log.12345. The `*` is to ignore all the extra characters the log might have. Might be something that just came up in my implementation but I think is a nice addition.